telethon/events/common.py

```diff
@@ -106,23 +106,26 @@ async def _resolve(self, client):
     def filter(self, event):
         """
         If the ID of ``event._chat_peer`` isn't in the chats set (or it is
-        but the set is a blacklist) returns `None`, otherwise the event.
+        but the set is a blacklist) returns `True`, otherwise `False`.
+        May also return awaitable which awaits to bool-able value.
```

Comment:
The problem with this is that it doesn't always return `True` or `False`. Just below there's a bare `return` (which implies `None` and not `False`).

Maybe just change the entire thing for something like the following:
```python
        """
        Returns a truthy value if the event passed the filter and should be
        used, or falsey otherwise. The return value may need to be awaited.
        """
```

Reply:
Yes, I looked for better word later. What do yoy think about truthy and falsy?